targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/CMakeLists.txt

```diff
@@ -1,20 +1,56 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-if("NRF52840_DK" IN_LIST MBED_TARGET_LABELS)
-    add_subdirectory(TARGET_NRF52840_DK)
+if ("ARDUINO_NANO33BLE" IN_LIST MBED_TARGET_LABELS)
+    target_include_directories(mbed-core
+        INTERFACE
+            TARGET_ARDUINO_NANO33BLE
+    )
+elseif("EP_AGORA" IN_LIST MBED_TARGET_LABELS)
+    target_include_directories(mbed-core
+        INTERFACE
+            TARGET_EP_AGORA
+    )
+    target_sources(mbed-core
+        INTERFACE
+            TARGET_EP_AGORA/ONBOARD_TELIT_ME910.cpp
```

Comment:
Does it not fail to build with this included?

Reply:
@0xc0170 good spot, curiously still builds without any issues.

Reply:
It builds fine because the content of the file is excluded since `MBED_CONF_NSAPI_PRESENT` is not defined.